src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.tsx

```diff
@@ -32,36 +32,61 @@ export const ProposalDataListItemStructure: React.FC<IProposalDataListItemStruct
 
     const ongoing = status === 'active' || status === 'challenged' || status === 'vetoed';
 
-    const publisherIsConnected = isConnected && connectedAddress?.toLowerCase() === publisher.address.toLowerCase();
-    const publisherLabel = publisherIsConnected
-        ? 'You'
-        : publisher.name ?? addressUtils.truncateAddress(publisher.address);
+    const parsedPublisher = Array.isArray(publisher)
+        ? publisher.map((p) => parsePublisher(p, isConnected, connectedAddress))
+        : [parsePublisher(publisher, isConnected, connectedAddress)];
+
+    const showParsedPublisher = parsedPublisher.length <= 3;
```

Comment:
We can define a `export const maxProposalPublishers = 3` const above the component and use it for the tests as well if we want to change it later on